Libraries/VtkVgCore/vtkVgBaseImageSource.h

```diff
@@ -100,21 +100,21 @@ class VTKVG_CORE_EXPORT vtkVgBaseImageSource : public vtkImageAlgorithm
   // Get image timestamp
   virtual vtkVgTimeStamp GetImageTimeStamp()
     {
-    return this->ImageTimeStamp;
+    return {};
```

Comment:
This caught my eye, thinking it could cause problems - but I probably should no better; it does in fact return a non-null (default) vtkVGTimeStamp?

Reply:
This is a [return](https://en.cppreference.com/w/cpp/language/return) by [list initialization](https://en.cppreference.com/w/cpp/language/list_initialization) (see (2) at first link).

Reply:
Addendum: I'm not sure if I need to be concerned that you mentioned "non-null"(?), but given a function that returns `T`, these three lines are generally equivalent:
```c++
  return {...args...};
  return T{...args...};
  return T(...args...);
```

(IIRC, the type name is occasionally necessary due to e.g. explicit constructors. Likewise, there can be differences if a constructor that takes a `std::initializer_list` is present. However, in *most* cases, the above are interchangeable.)

In this specific instance, `return {}` is equivalent to `return vtkVgTimeStamp()`.